Cargo.toml

```diff
@@ -12,14 +12,20 @@ repository = "https://github.com/datafuselabs/opendal"
 version = "0.11.3"
 
 [package.metadata.docs.rs]
-all-features = true
+features = ["compres", "retry", "serde", "services-hdfs", "services-http"]
 
 [features]
+default = ["epoll"]
 compress = ["async-compression"]
 retry = ["backon"]
 serde = ["time/serde"]
 services-hdfs = ["hdrs"]
 services-http = ["radix_trie"]
+# the io features, by default, we use `epoll`
+# to enable `iouring` we need to disable the `default` feature
+epoll = ["nuclei/epoll"]
+iouring = ["nuclei/iouring"]
```

Comment:
I prefer `services-fs-iouring` to indicate it's a feature for services fs.

Reply:
Let me start with epoll, iouring will probably be done in the future.